src/Package/Impl/DataInspect/EvaluationWrapper.cs

```diff
@@ -148,5 +153,16 @@ public sealed class EvaluationWrapper : RSessionDataObject, IIndexedItem {
             return false;
         }
         #endregion
+
+        private bool ComputCsvAvailability(DebugValueEvaluationResult evaluation) {
```

Comment:
Is there any particular reason why this is separate from `ComputeDetailAvailability` (which should probably be renamed to something like `ComputeGridAvailability`)? It would appear that if we can open something in the grid, it should be exportable to Excel, and vice versa.